runtime/doc/lsp.txt

```diff
@@ -878,6 +878,18 @@ buf_diagnostics_underline({bufnr}, {diagnostics})
 buf_diagnostics_virtual_text({bufnr}, {diagnostics})
                 TODO: Documentation
 
+                                 *vim.lsp.util.buf_diagnostics_signs()*
+buf_diagnostics_signs({bufnr}, {diagnostics})
+                Place signs for each diagnostic in the sign column.
+                Sign characters can be customized with the following options:
+>
+let g:LspDiagnosticsErrorSign = 'E'
+let g:spDiagnosticsWarningSign = 'W'
```

Comment:
Typo: should be ` g:LspDiagnosticsWarningSign`

Reply:
Thanks!